README.md

```diff
@@ -12,8 +12,16 @@ week | chapters   | pages | comments
 -----|------------|-------|---------
 04/11/16  | Elixir Hack Night | |
 04/25/16  | Elixir Hack Night | |
-05/09/16  | Intro and 1 | ? |
-05/16/16  | Chapter 2 | |
+05/09/16  | Chapter 1 | 12 |
+05/16/16  | Chapter 2 | 21 |
+05/23/16  | Chapter 3 & 4 | 29 |
+05/30/16  | Chapter 5 | 20 |
```

Comment:
I'm going to be out of town 5/30, 6/6 and the week of 6/13.

I will be here from 6/7 - 6/11, so if we could meet any of those days, I'd be able to attend that week.